apps/storybook/src/overlays/Menu.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import {
+  Button,
+  Menu,
+  MenuContent,
+  MenuItem,
+  MenuTrigger,
+} from "@optiaxiom/react";
+import { IconStar } from "@tabler/icons-react";
+
+const meta: Meta<typeof Menu> = {
+  component: Menu,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof MenuContent>;
+
+export const Basic: Story = {
```

Comment:
Can we add an interaction test for this?

Reply:
addressed. can you check back?